week-5-serving-streamlit/app.py

```diff
@@ -0,0 +1,70 @@
+import os
+from typing import List, Text, Tuple
+from pathlib import Path
+import torch
+from transformers import BertTokenizer
+from model import RegressionModel
+import gdown
+import streamlit as st
+
+
+MODEL_URL = "https://drive.google.com/file/d/1M5SFB5cYS7Q3oLpkEQVGERXtquwPxury/view?usp=sharing"
+MODEL_SAVE_PATH = "./weights/bert-base-uncased-regression-weights.pth"
+
+
+@st.cache
+def init_model() -> Tuple[torch.nn.Module, torch.nn.Module, torch.device]:
+    folder = Path("./weights")
+    weights = Path(MODEL_SAVE_PATH)
+    if not folder.exists():
+        os.mkdir("./weights")
```

Comment:
should we use `folder.mkdir()`?

Reply:
Definitely. I just used to `os` instead of `Path`. It's like bad habit, you know.